Plugins/InstallSafeDITool/Shared.swift

```diff
@@ -0,0 +1 @@
+../Shared.swift
```

Comment:
this is how I'm sharing code between Plugins, which can't have shared dependencies.